Packs/PhishUp/Integrations/PhishUp/README.md

```diff
@@ -0,0 +1,81 @@
+PhishUp prevents phishing attacks, protects your staff and your brand with AI
+
+## Configure PhishUp on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for PhishUp.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API KEY |  | True |
+    | Incident type |  | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | PhishUp Playbook Actions | If there is any Phishing activity in mail, what should PhishUp do? | True |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### phishup-investigate-url
+***
+Single url investigation
+
+
+#### Base Command
+
+`phishup-investigate-url`
```

Comment:
Please update Readme to include the updated commands (can be copied from the description)